cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -38,11 +38,11 @@
     {% set video_id = options.video.id | default('') %}
 
     {# TODO: Replace with django method for getting video id if youtube and urlparams #}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2086/commits/1b34db1baa07ea54473a14138bfa679ddedfe670